src/components/Selectbox/SelectboxAnchor.jsx

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import {themr} from 'react-css-themr';
+import {PURE} from 'dx-util/src/react/pure';
+import Button from '../Button/Button.jsx';
+import classnames from 'classnames';
+
+export const SELECTBOX_ANCHOR = Symbol('SelectboxAnchor');
+
+export const ANCHOR_THEME = {
+	container: React.PropTypes.string,
+	text: React.PropTypes.string,
+	content: React.PropTypes.string,
+	content_hasCaret: React.PropTypes.string,
+	caret: React.PropTypes.string,
+	caret_isReversed: React.PropTypes.string
+};
+
+@PURE
+@themr(SELECTBOX_ANCHOR)
+export default class SelectboxAnchor extends React.Component {
+	static propTypes = {
+		...Button.PropTypes,
+		theme: React.PropTypes.shape(ANCHOR_THEME),
+		isDisabled: React.PropTypes.bool,
```

Comment:
`isDisabled` and `isPrimary` are possibly already defined in `Button.propTypes` - check